R/add_stylecss.R

```diff
@@ -0,0 +1,40 @@
+#' add_stylecss
+#' @description Add PHS template style.css file to chosen directory.
+#'
+#' @param path String: path to add style.css file. If left blank, will prompt the user within R Studio
+#'
+#' @return Adds style.css file to directory.
+#' @export
+#' @examples
+#' \dontrun{
+#' add_stylecss()
+#' }
+add_stylecss <- function(path = rstudioapi::selectDirectory(caption = "Select folder to add style.css")) {
+  if (is.null(path)) {
+    return(message("style.css file not added."))
```

Comment:
This is a bit confusing to me, since `message` will always return `NULL`

```suggestion
    message("style.css file not added.")
    return(NULL)
```

Reply:
I basically copied stuff over from the `add_gitignore()` function but don't thing we need `return()` at all

Reply:
I always prefer to have an explicit `return()` 😄 but I'm just a stickler!